### ISSUE TITLE
Drush 11.x-dev Incompatible with drupal/core-dev:9

### ISSUE THREAD

**Issue by mikemadison13** [drush-ops/drush#5277]
**Describe the bug**
As of https://github.com/drush-ops/drush/commit/888bea7bf7cfb7a7b30212bf9cfa021fddbe96de Drush 11.x-dev is no longer compatible with the drupal/core require-dev dependencies. This is because Drupal 9.4.x requires symfony/file-system 4.4.x and Drush now requires 5.4.x. 

This is an issue for any project that pulls in the drupal/core-dev package for automated testing purposes. 

I've opened https://www.drupal.org/project/drupal/issues/3315217 on D.O to try and get the dependency changed there, but this feels like a breaking change on the drush side that could/should be addressed. 

**To Reproduce**
1. Update to the current drush 11.x head
2. Attempt to install both drush 11.x-dev and drupal/core-dev for phpunit testing

**Expected behavior**
Drush 11 should remain compatible with Drupal 9.x.

**Actual behavior**
```
Problem 1
    - Root composer.json requires acquia/blt 13.x-dev -> satisfiable by acquia/blt[13.x-dev (alias of dev-main)].
    - drupal/core-dev 9.4.x-dev requires symfony/filesystem ^4.4 -> satisfiable by symfony/filesystem[v4.4.0-BETA1, ..., 4.4.x-dev].
    - You can only install one version of a package, so only one of these can be installed: symfony/filesystem[v2.3.0, ..., 2.8.x-dev, v3.0.0-BETA1, ..., 3.4.x-dev, v4.0.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev, v6.0.0-BETA1, ..., 6.2.x-dev].
    - drush/drush 11.x-dev requires symfony/filesystem ^5.4 -> satisfiable by symfony/filesystem[v5.4.0-BETA1, ..., 5.4.x-dev].
    - acquia/blt dev-main requires drush/drush ^11 -> satisfiable by drush/drush[11.x-dev].
    - acquia/blt 13.x-dev is an alias of acquia/blt dev-main and thus requires it to be installed too.
    - Root composer.json requires drupal/core-dev 9.4.x-dev -> satisfiable by drupal/core-dev[9.4.x-dev].
 ``` 

**Workaround**
The current workaround is to pin drush to the most recent release and avoid the dev branch.

**Comment by weitzman** [comment]
Yes, we got a report at https://github.com/drush-ops/drush/pull/5264#issuecomment-1277914069 and many other places. @greg-1-anderson is working on a PR which brings in a private copy of the Path class for Symfony 4 sites.

**Comment by mikemadison13** [comment]
awesome, thanks!

**Comment by weitzman** [comment]
Fixed in #5281 and related PRs. A release is forthcoming.
